guides/common/modules/ref_options-in-managed-dhcpv4.adoc

```diff
@@ -1,23 +1,22 @@
-[id="DHCP_Options_{context}"]
-= {Project} and DHCP options
+[id="options-in-managed-dhcpv4"]
+= Options in managed DHCPv4
 
-{Project} manages DHCP reservations through a DHCP {SmartProxy}.
-{Project} also sets the `next-server` and `filename` DHCP options.
+When {Project} manages the DHCP service and can update the DHCP configuration, {Project} sets the `next-server` and `filename` DHCP options.
 
-.The next-server option
+`next-server`::
 The `next-server` option provides the IP address of the TFTP server to boot from.
 This option is not set by default and must be set for each TFTP {SmartProxy}.
-You can use the `{foreman-installer}` command with the `--foreman-proxy-tftp-servername` option to set the TFTP server in the `/etc/foreman-proxy/settings.d/tftp.yml` file:
-
+You can use the `{foreman-installer}` command with the `--foreman-proxy-tftp-servername` argument to set the TFTP server in the `/etc/foreman-proxy/settings.d/tftp.yml` file:
++
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} --foreman-proxy-tftp-servername _1.2.3.4_
 ----
-
++
 Each TFTP {SmartProxy} then reports this setting through the API and {Project} can retrieve the configuration information when it creates the DHCP record.
-
++
 When the PXE loader is set to `none`, {Project} does not populate the `next-server` option into the DHCP record.
-
++
 If the `next-server` option remains undefined, {Project} uses reverse DNS search to find a TFTP server address to assign, but you might encounter the following problems:
```

Comment:
I know this was already there, but this behavior has changed a long time ago. In Foreman 2.0 there was https://github.com/theforeman/foreman/commit/60c03b6a2e92c432ee66f2b27da634954c1b646c which deprecated the old reverse DNS lookup for Smart Proxies incapable of a new feature (for n-1 support). In Foreman 2.5 this deprecated behavior was removed with https://github.com/theforeman/foreman/commit/f3231f62e600db1a1616c1bdf9c5fa8aed5e27ee.

The way it now works is:
* It calls the Smart Proxy API to retrieve the serverName (as specified using `--foreman-proxy-tftp-servername` in the installer)
* If this isn't returned, it uses the hostname of the Smart Proxy

It is up to the DHCP provider server to deal with a hostname and it depends on the implementation how this exactly works. For ISC DHCP the Smart Proxy now resolves the address:
https://github.com/theforeman/smart-proxy/blob/7060ce3e3951f070475e0c977fd3bf93e335d810/modules/dhcp_common/isc/omapi_provider.rb#L203-L215

And in reviewing this PR I've realized we should really use the option to set the tftp server name and let ISC DHCP deal with it.

Reply:
@ekohl Thank you. I've created a separate PR so that we can cherry pick it.
Please review https://github.com/theforeman/foreman-documentation/pull/3540